2020-04-07 21:19:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "recipe_id" not found [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3148)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3247)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:310)
	at main.domain.Logic.testDatabase(Logic.java:83)
	at main.ui.TempTextUI.start(TempTextUI.java:46)
	at main.Main.main(Main.java:35)
